components/QuestionAdd/ReposSelector.tsx

```diff
@@ -300,17 +410,12 @@ const RepositorySelectionSystem = ({
               {selectedCount !== 0 && (
                 <div>
                   {selectedArray.map(repo => (
-                    <div key={repo.id} className={styles.item}>
+                    <div key={repo.id} className={styles.item} id={String(repo.id)}>
                       <div className={styles.itemHeader}>
                         <div className={styles.itemContent}>
-                          <div className={styles.itemTitle}>{repo.name}</div>
-                          <div className={styles.itemMeta}>
-                            <span className={`${styles.itemBadge} ${repo.access?.toLowerCase() === 'open' ? styles.open : ''}`}>
-                              {repo.access}
-                            </span>
-                            <span className={styles.itemBadge}>{repo.identifier}</span>
+                          <div className={styles.itemTitle}>{repo.name}
                             <a
-                              href={repo.url}
+                              href={repo.uri}
```

Comment:
you'd want to use the website here instead of the uri. The uri makes it machine-actionable but the user isn't going to care about the re3data landing page. They'd be more interested in being taken out to the repo's homepage